services/mock-service/src/fsp-integration/safaricom/safaricom.mock.service.ts

```diff
@@ -165,10 +148,10 @@ export class SafaricomMockService {
       };
     } else if (mockScenario === MockScenario.errorOnCallback) {
       response = {
-        Result: otherFailureStatus.Result,
+        Result: callbackErrorResponse.Result,
       };
     } else if (mockScenario === MockScenario.errorOnCallbackForTimeOut) {
-      response = timeoutCallbackResponse;
+      response = transferDto;
```

Comment:
maybe add a comment here explaining that safaricom has explained us via email that the initial request body is returned as response here.